topbeat/etc/fields.yml

```diff
@@ -37,6 +34,17 @@ env:
         The hostname as returned by the operating system on which the Beat is
         running.
 
+    - name: tags
```

Comment:
We could store this part in libbeat and fetch it from there as it is the same across all beats.